governance.md

```diff
@@ -22,18 +28,20 @@ website (https://pangeo-data.org and https://pangeo.io), Pangeo-JupyterHub
 deployments (https://pangeo.pydata.org and https://pangeo.informaticslab.co.uk),
 and the Pangeo-Binder (https://binder.pangeo.io).
 
-The Project is developed by a team of distributed developers, called
+
+## Contributors
+
+The Project is maintained by a distributed team of people, called
 Contributors. Contributors are individuals who have contributed to code, code
-reviews, documentation, design, and infrastructure, or to mailing lists, chats,
+reviews, documentation, design, infrastructure, mailing lists, chats,
 community help and community building, education and outreach, or other work in
 relation to one or more Project repositories. Anyone can be a Contributor.
-Contributors can be affiliated with any legal entity or none. Contributors
```

Comment:
Removed a sentence here because it mainly repeats information already present above in this paragraph, but without acknowledging contributions outside of Github/mailing lists (like community building, outreach, or running workshops). The additional idea of "we do our stuff on Github, you should use it!" is communicated by the contributor list linked below and elsewhere in documentation about Pangeo.